plugins/fortinet_fortiproxy.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "fortinet_fortiproxy"
```

Comment:
is the priactice to have _ in there? I think a space is better.

Reply:
Okay, resolved.